Makefile

```diff
@@ -283,6 +285,11 @@ endif
 			rm -rf $(DESTDIR)$(pkglibdir)/modules.d/[0-9][0-9]$${i}; \
 		done \
 	fi
+ifeq ($(ARCH),s390x)
+	for f in cio_ignore cms dasd dasd_mod dcssblk zfcp zipl znet; do \
+		rm -rf $(DESTDIR)$(pkglibdir)/modules.d/[0-9][0-9]$${i}; \
```

Comment:
I would drop `-f` since we expect those directories to be present. That way we would notice if we forgot to update the list here.